packages/contracts/contracts/package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
why two package.json in this package?

Reply:
this makes easy to release just the contracts and avoid carrying on a huge bundle of configs & other dev env packages. This is the default project structure when using `hardhat`.